hbase-server/src/test/java/org/apache/hadoop/hbase/master/TestAssignmentManagerOnCluster.java

```diff
@@ -598,6 +601,68 @@ public void testCloseFailed() throws Exception {
     }
   }
 
+  /**
+   * This tests region close with exponential backoff
+   */
+  @Test(timeout = 60000)
+  public void testCloseRegionWithExponentialBackOff() throws Exception {
+    String table = "testCloseRegionWithExponentialBackOff";
```

Comment:
nit: We use TestTableName.getTableName() that automatically gets the TableName with the running test name.